modules/indexer/issues/bleve.go

```diff
@@ -236,6 +239,7 @@ func (b *BleveIndexer) Search(keyword string, repoIDs []int64, limit, start int)
 	for _, repoID := range repoIDs {
 		repoQueriesP = append(repoQueriesP, numericEqualityQuery(repoID, "RepoID"))
 	}
+	index, _ := strconv.ParseInt(keyword, 10, 64)
```

Comment:
Maybe keyword is a string?

Reply:
does it matter?
I didn't notice a difference in behavior.
How would I dynamically add the query if keyword is a string and the parser errors?